src/ImageSharp/Formats/Tiff/TiffMetadata.cs

```diff
@@ -31,6 +44,14 @@ public TiffMetadata()
     /// </summary>
     public TiffFormatType FormatType { get; set; }
 
+    /// <summary>
+    /// Gets or sets the frames.
+    /// </summary>
+    /// <value>
+    /// The frames.
+    /// </value>
+    public IReadOnlyList<TiffFrameMetadata> Frames { get; set; } = Array.Empty<TiffFrameMetadata>();
```

Comment:
If you look at how this is done with our Gif format you will see that the individual frame metadata is not a property of the main image one. 

They're stored in a dictionary within individual `ImageFrameMetadata` instances.

Reply:
Just realized what you mean by this. 

I've pulled your code down and will work on a refactor to add the info to `ImageInfo`

Reply:
> I've pulled your code down and will work on a refactor to add the info to `ImageInfo`

yes, this way looks more mature